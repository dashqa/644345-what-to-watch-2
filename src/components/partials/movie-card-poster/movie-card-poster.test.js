import React from 'react';
import renderer from 'react-test-renderer';
import MovieCardPoster from './movie-card-poster';

const props = {
  name: `We need to talk about Kevin`,
  posterImage: `http://image.com/image.png`,
  classMods: `card-poster`,
};

it(`Movie card poster correctly renders after relaunch`, () => {
  const tree = renderer.create(<MovieCardPoster {...props}/>).toJSON();
  expect(tree).toMatchSnapshot();
});
