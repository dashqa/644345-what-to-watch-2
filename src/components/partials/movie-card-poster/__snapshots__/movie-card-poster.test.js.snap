// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Movie card poster correctly renders after relaunch 1`] = `
<div
  className="movie-card__poster card-poster"
>
  <img
    alt="We need to talk about Kevin"
    height="327"
    src="http://image.com/image.png"
    width="218"
  />
</div>
`;
